pop-api/integration-tests/src/lib.rs

```diff
@@ -82,8 +82,7 @@ fn bare_call(
 
 // Deploy, instantiate and return contract address.
 fn instantiate(contract: &str, init_value: u128, salt: Vec<u8>) -> AccountId32 {
-	let (wasm_binary, _) =
-		load_wasm_module::<Runtime>(contract).expect("could not read .wasm file");
+	let wasm_binary = load_wasm_module::<Runtime>(contract).expect("could not read .wasm file");
```

Comment:
If this is the only usage, we dont need a whole function when `std::fs::read(contract).expect("could not read .wasm file")` would be enough.